src/libraries/System.Private.Xml/tests/TrimmingTests/System.Private.Xml.TrimmingTests.proj

```diff
@@ -2,12 +2,22 @@
   <Import Project="$([MSBuild]::GetPathOfFileAbove(Directory.Build.props))" />
 
   <ItemGroup>
-    <TestConsoleAppSourceFiles Include="XmlSchema.Write.cs" />
-    <TestConsoleAppSourceFiles Include="XmlSerializer.Deserialize.cs" />
+    <TestConsoleAppSourceFiles Include="XmlSchema.Write.cs">
+      <!-- What the test is testing is RUC and there was a half-hearted attempt to make it safe, but it isn't: https://github.com/dotnet/runtime/blob/5f067ce8b50087e032d13a1b97ae5ec39fc54739/src/libraries/System.Private.Xml/src/System/Xml/Schema/XmlSchema.cs#L134-L137 -->
+      <NativeAotIncompatible>true</NativeAotIncompatible>
+    </TestConsoleAppSourceFiles>
+    <TestConsoleAppSourceFiles Include="XmlSerializer.Deserialize.cs">
+      <NativeAotIncompatible>true</NativeAotIncompatible>
+    </TestConsoleAppSourceFiles>
     <TestConsoleAppSourceFiles Include="XmlSerializer.Deserialize.SealerOpt.cs"
                                ExtraTrimmerArgs="--enable-opt sealer" />
```

Comment:
The ExtraTrimmerArgs is kind of odd in the context of native AOT. Should this test be disabled too?

Reply:
Sealing is an optimization that native AOT does automatically so it doesn't seem too irrelevant. Native AOT's version should not really be observable.